libsolidity/analysis/SyntaxChecker.cpp

```diff
@@ -263,6 +263,13 @@ bool SyntaxChecker::visit(UnaryOperation const& _operation)
 
 bool SyntaxChecker::visit(InlineAssembly const& _inlineAssembly)
 {
+	if (yul::PCFinder::containsPC(_inlineAssembly.dialect(), _inlineAssembly.operations()))
```

Comment:
Please use `AsmAnalyzer::warnOnInstructions` - this is where all the other instruction warnings are.